deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```diff
@@ -136,9 +136,9 @@ spec:
             {{- toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
-      {{- with .Values.cainjector.nodeSelector }}
```

Comment:
This conditional should retain to allow users to omit node selectors.

Reply:
Ah. You are right. Thanks. I have pushed an update to fix this.

Reply:
I suggest you keep the `with` and use `.`  as input to the loop. No need to repeat the value paths and less changes in the PR. 👍

Reply:
Thanks, looks good now.